src/App.js

```diff
@@ -1,26 +1,49 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import Api from './Api';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: []
+    };
+  }
+
+  componentDidMount() {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+    .then(response => response.json())
+    .then(data => this.setState({ movies: data.movies }));
+  }
+
+  separateMovies() {
+    const moviesToDisplay = this.state.movies.map((movie, index)=> {
+      return (
+        <section key={index}>
+          <img src={movie.poster_path}></img>
+          <h2>{movie.title}</h2>
+          <h3>{movie.average_rating}</h3>
+          <h3>{movie.release_date}</h3>
+        </section>
```

Comment:
might look at moving this out into another file in the future

Reply:
Great idea, I think we should do that before we get too far